mlir/lib/Driver/CompilerDriver.cpp

```diff
@@ -404,9 +387,7 @@ LogicalResult QuantumDriverMain(const CompilerOptions &options, CompilerOutput &
     MLIRContext ctx(registry);
     ctx.printOpOnDiagnostic(true);
     ctx.printStackTraceOnDiagnostic(options.verbosity >= Verbosity::Debug);
-    // TODO: FIXME:
-    // Let's try to enable multithreading. Do not forget to protect the printing.
```

Comment:
There might be issues with file name counter, if I remeber it correctly. Could you please check?